psi4/driver/procrouting/scf_proc/subclass_methods.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+from psi4 import core
+
+def _RHF_orbital_gradient(self, save_fock: bool, max_diis_vectors: int) -> float:
```

Comment:
general info to readers: the Sphinx docs can now process type annotations, so they're not added in vain. the Sphinx docs now also hate the old Parameters\n verbose : int style in-docstring type annotation, so those're worth moving up into the signature.